x509/codesigning_cert_validations.go

```diff
@@ -145,35 +149,25 @@ func validateCodeSigningKeyUsagePresent(cert *x509.Certificate) error {
 	return nil
 }
 
-func validateCodeSigningExtendedKeyUsage(cert *x509.Certificate, expectedEku x509.ExtKeyUsage) error {
+func validateCodeSigningExtendedKeyUsage(cert *x509.Certificate) error {
 	if len(cert.ExtKeyUsage) == 0 {
 		return nil
 	}
 
 	excludedEkus := []x509.ExtKeyUsage{
-		x509.ExtKeyUsageAny,
 		x509.ExtKeyUsageServerAuth,
 		x509.ExtKeyUsageClientAuth,
 		x509.ExtKeyUsageEmailProtection,
 		x509.ExtKeyUsageTimeStamping,
 		x509.ExtKeyUsageOCSPSigning,
 	}
 
-	var hasExpectedEku bool
 	for _, certEku := range cert.ExtKeyUsage {
-		if certEku == expectedEku {
-			hasExpectedEku = true
-			continue
-		}
```

Comment:
What's the reason of removing this check?

Reply:
Because we removed `expectedEku`, originally, this value is always '0'. To keep the logic unchanged, we could remove `x509.ExtKeyUsageAny` from the `excludedEkus` and only validate `excludedEkus`.